packages/babel-generator/test/fixtures/decorators-legacy/ts-class-modifier-with-retainlines/output.js

```diff
@@ -19,4 +19,5 @@ class A {
 
   private [cm]()
 
-  {}}
+  {}
```

Comment:
Whops :grimacing:

Reply:
I also thought it was a throwback when I first saw it, but on closer inspection it wasn't.😃

![image](https://user-images.githubusercontent.com/30521560/192666307-acf8eedf-799a-4324-8a22-5fcba065b9bb.png)
![image](https://user-images.githubusercontent.com/30521560/192666324-104b1d2f-2fcd-44ea-8071-dd0df8928d1a.png)